Source/UI/Coordinators/PasswordCoordinator.swift

```diff
@@ -8,6 +8,7 @@ import UIKit
 
 private struct Constants {
```

Comment:
add: `private init() { }` to avoid possibility to create an instance of `Constants`